src/objdump.py

```diff
@@ -24,6 +25,12 @@
 re_int_full = re.compile(r"\b[0-9]+\b")
 
 
+@dataclass
+class Line:
+    row: str
```

Comment:
include the mnemonic here, since we already parse it out and use it in the scorer